include/csoundCore.h

```diff
@@ -559,13 +559,15 @@ typedef struct CORFIL {
     double   offbet;
     /* Time to turn off event, in seconds (negative on indef/tie) */
     double   offtim;
-    /* Python namespace for just this instance. */
-    void    *pylocal;
```

Comment:
I assume removing this will break the Python opcodes. Noting this here that those may need to be updated for CS7.